lib/addons/cloud-watch-insights/index.ts

```diff
@@ -0,0 +1,75 @@
+import {Construct} from 'constructs';
+import * as eks from "aws-cdk-lib/aws-eks";
+import * as iam from "aws-cdk-lib/aws-iam";
+import {ClusterInfo} from "../../spi";
+import {conflictsWith, supportsALL} from "../../utils";
+import {CoreAddOn, CoreAddOnProps} from "../core-addon";
+import {ebsCollectorPolicy} from "./iam-policy";
+
+/**
+ * Configuration options for AWS Container Insights add-on.
+ */
+export type CloudWatchInsightsAddOnProps = Omit<CoreAddOnProps, "saName" | "addOnName" | "version"> & {
+  /**
+   * Gives CloudWatch agent access to EBS performance systems by adding an IAM role as defined here:
+   * https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/install-CloudWatch-Observability-EKS-addon.html#install-CloudWatch-Observability-EKS-addon-configuration
+   */
+  ebsPerformanceLogs?: boolean,
+  /**
+   * Custom CloudWatch Agent configuration, specifics can be found here:
+   * https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/install-CloudWatch-Observability-EKS-addon.html#install-CloudWatch-Observability-EKS-addon-configuration
+   */
```

Comment:
[Container Logs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/install-CloudWatch-Observability-EKS-addon.html#install-CloudWatch-Observability-EKS-addon-configuration:~:text=Optional)%20Additional%20configuration-,Container%20logs,-By%20default%2C%20the) option is missing. This means we can have a conflicts with CW Logs addon?